xml/System.Threading/Timer.xml

```diff
@@ -868,8 +868,8 @@ Sub TimerCallback(state As Object)
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Releases all resources used by the current instance of <see cref="T:System.Threading.Timer" />.</summary>
+        <returns>A <see cref="T:System.Threading.Tasks.ValueTask" /> that completes when all work associated with the timer has ceased.</returns>
         <remarks>To be added.</remarks>
```

Comment:
Based on @rpetrusha suggestion above, the exception mentioned in the `<returns>` could be moved here:

```suggestion
        <remarks>To be added.</remarks>
        <exception cref="T:System.ArgumentNullException"><paramref name="callback" /> is <see langword="null" />.</exception>
```

Reply:
What throws NotSupportedException?

Reply:
My bad. I added the suggestion in the wrong location. I meant to add it for UnsafeQueueUserWorkItem, not DisposeAsync. I'm resolving this comment and adding the suggestion in the right place.